### PR TITLE
spotpass: Add npfl to regex for 3DS

### PR DESCRIPTION
**Pull request by MisterSheeple**
A small number of tasks were discovered from npfl URLs that were not otherwise present. This pull adds npfl to the regex to ensure that those tasks are not missed.

### CODE REVIEW

**Comment by MisterSheeple** [comment]
~~Also, I should note that I haven't spotted any NPFL urls for any Wii U dumps, so I didn't add it there. But if you think it should be used there as well, it's probably fine to just add it anyway.~~

Never mind, I see now that the Wii U regex works entirely differently.

File: spotpass/read-boss-db-3ds.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const fs = require('fs-extra');
 const apps = require('./ctr-boss-apps.json');
 
-const NPDL_REGEX = /npdl\.(?:cdn|c\.app)\.nintendowifi\.net\/p01\/nsa\/([A-z0-9]{16})\/(\w*)/g;
+const NPDL_REGEX = /(?:npdl|npfl)\.(?:cdn|c\.app)\.nintendowifi\.net\/p01\/(?:nsa|filelist)\/([A-z0-9]{16})\/(\w*)/g;
```
Comment (jonbarrow) [inline spotpass/read-boss-db-3ds.js]: Only one thing. Either this variable needs to be renamed, or the NPFL search needs to be a separate regex. Either way works

**Review by jonbarrow — CHANGES_REQUESTED** [review]
Good catch!

File: spotpass/read-boss-db-3ds.js (hunk 1 above)
Comment (MisterSheeple) [reply spotpass/read-boss-db-3ds.js]: I just renamed it to `REGEX`. Hope that works.

Comment (jonbarrow) [reply spotpass/read-boss-db-3ds.js]: Works for me :+1:

**Pull request merged**